site.properties

```diff
@@ -4,7 +4,7 @@
 
 # Build tagging
 version.edition = @version.edition@
-version.number  = 7.19
+version.number  = 7.20
```

Comment:
I think it should be 7.20-development.
I am fine for the other changes

Reply:
I'll change it to 7.20-SNAPSHOT, since that's the GAV version coordinate.